pkg/container-collection/options.go

```diff
@@ -36,6 +36,31 @@ import (
 	"github.com/kinvolk/inspektor-gadget/pkg/runcfanotify"
 )
 
+func enrichContainerWithContainerData(containerData *runtimeclient.ContainerData, container *Container) {
+	// Runtime
+	container.ID = containerData.ID
+	container.Runtime = containerData.Runtime
+
+	// Kubernetes
+	container.Namespace = containerData.PodNamespace
+	container.Podname = containerData.PodName
+	container.PodUID = containerData.PodUID
+
+	// Notice we are temporarily using the runtime container name as the
+	// Kubernetes container name because the Container struct doesn't have that
+	// field, and we don't support filtering by runtime container name yet.
+	container.Name = containerData.Name
```

Comment:
Take into account that this PR is not introducing this change, this is already the current implementation.